pom.xml

```diff
@@ -130,13 +130,17 @@
     <!-- maven-compiler-plugin -->
     <maven.compiler.target>1.7</maven.compiler.target>
     <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.testTarget>1.7</maven.compiler.testTarget>
```

Comment:
this should be reference to maven.compiler.target so it can be controlled with single property by default and overriden if needed.